lib/screens/signup_screen.dart

```diff
@@ -18,7 +18,7 @@ final signupRepositoryProvider =
     Provider<SignupRepository>((ref) => SignupRepository());
 
 class SignupRepository {
-  var url = Uri.parse("http://192.168.0.44:8888/api/signup");
+  var url = Uri.parse("http://192.168.1.104/api/signup");
```

Comment:
The base URL should come from a central setting. Be that an env variable, static const etc. That way you can change the URL of all queries at once.

Reply:
Thanks for the suggestion. I agree with you. Also, The Sign Up Screen doesn't use the functions in `UserService` and `HttpService`. I think there will be another updating the signup screen including the URL part.